src/Time/TimeSteppers/TimeStepper.hpp

```diff
@@ -164,6 +197,15 @@ class LtsTimeStepper : public TimeStepper::Inherit {
   /// Substep LTS integrators are not supported, so this is always 1.
   uint64_t number_of_substeps() const noexcept final { return 1; }
 
+  /// Substep LTS integrators are not supported, so this is always 1.
+  uint64_t number_of_substeps_for_error() const noexcept final { return 1; }
```

Comment:
should this have an `ERROR` in it? I guess the `number_of_substeps` doesn't either, so unclear to me. I assume we can't `ERROR` because generic code calls this and `1` is a valid answer, is that correct @wthrowe?

Reply:
Correct.

Reply:
Thanks!